lib/repl.js

```diff
@@ -319,9 +320,20 @@ function REPLServer(prompt,
   self.outputStream = output;
 
   self.resetContext();
-  self.bufferedCommand = '';
   self.lines.level = [];
 
+  self.clearBufferedCommand = function clearBufferedCommand() {
```

Comment:
IMHO it would be better to add `clearBufferedCommand ` to `REPLServer.prototype`.

About `Object.defineProperty(this, 'bufferedCommand'` I'm not sure since that will change the `.hasOwnProperty('bufferedCommand')`